packages/osd-ui-shared-deps/webpack.config.js

```diff
@@ -150,6 +170,15 @@ exports.getWebpackConfig = ({ dev = false } = {}) => ({
   resolve: {
     alias: {
       moment: MOMENT_SRC,
+      // Add aliases for Monaco 0.30.1 paths
```

Comment:
It looks like all these alias resolve to the same path. If so are these needed?

Reply:
nice catch. will remove.